### PR TITLE
docs: update feature flag to include addMissingOwerFields

### PR DESCRIPTION
**Pull request by yuth**
update feature flag to include graphQLTransformer.addMissingOwnerFields

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by renebrandel — APPROVED** [review]

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+        "values": [
+          {
+            "value": "true",
+            "description": "inserts the owner field from auth rules when its missing in type",
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "don't insert the owner field automatically",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:19]: Doesnt*

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+    "features": {
+      "addMissingOwnerFields": {
+        "description": "Automatically add owner field to when owner fields are not in the type",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.30.0",
+        "values": [
+          {
+            "value": "true",
+            "description": "inserts the owner field from auth rules when its missing in type",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:13]: Capitalize Inserts

**Review by renebrandel — CHANGES_REQUESTED** [review]

**Review by renebrandel — APPROVED** [review]

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 3]
```diff
@@ -1,2 +1,27 @@
 {
+  "graphQLTransformer": {
+    "description": "Feature Flag related to GraphQL Transformer",
+    "features": {
+      "addMissingOwnerFields": {
+        "description": "Automatically add owner field to when owner fields are not in the type",
```
Comment (attilah) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json]: the 'to' seems to be an extra, without it: "Automatically add owner field when owner fields are not in the type"

**Review by attilah — APPROVED** [review]
one nit comment

**Review by attilah — APPROVED** [review]

**Pull request merged**